source/simulator/stokes_matrix_free.cc

```diff
@@ -61,7 +61,7 @@ namespace aspect
       add_constraint(const std::array<types::global_dof_index,dim> &dof_indices,
                      const Tensor<1, dim> &constraining_vector,
                      AffineConstraints<double> &constraints,
-                     const double inhomogeneity = 0)
+                     const double inhomogeneity)
```

Comment:
Why did you remove the default value here, but kept it above for mesh_deformation? There may be a reason, I just want to make sure it was not an accident.